1. Array/weird_sorting/WeirdSortingArnab.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * WeirdSortingArnab.java
+ * Given an array of integers containing 7,14 and 21.Sort them in a non-decreasing manner with O(n) time and space 
+   complexity
+ * Description:-
+ * Count occurences of 7,14 and 21 in 3 separate variables.Then just add each value in the same array starting from 7
+ * When 7 ends,add 14 and when 14 ends fill up rest with 21.
+ * Time Complexity-O(n) Extra Space Complexity-O(1)
+ * @author [codebook-2000] (https://github.com/codebook-2000)
+ */
+ 
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+
+class Weird_Sorting
+{
+    static BufferedReader buf=new BufferedReader(new InputStreamReader(System.in));
+    static int arr[];
+    static void solve() throws java.lang.Exception
+    {
+        int n=Integer.parseInt(buf.readLine());
```

Comment:
Take input from main() method into an array and pass it as an argument into solve() method